srcpkgs/rpi-usbboot/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'rpi-usbboot'
+_commit=1e651a88f0313a5ca50b12a9992dbce8930fe200
+pkgname=rpi-usbboot
+version=20220815.145439
+revision=1
+wrksrc="${pkgname/rpi-/}-${_commit}"
+build_style=gnu-makefile
+#make_dirs="/var/log/dir 0755 root root"
```

Comment:
Please remove commented lines.

Reply:
usually I remember to remove those, not sure why I forgot this time :D